pkg/apis/servicecatalog/validation/serviceclass.go

```diff
@@ -30,7 +30,7 @@ import (
 // validateServiceClassName is the validation function for ServiceClass names.
 var validateServiceClassName = apivalidation.NameIsDNSSubdomain
 
-const guidFmt string = "[a-zA-Z0-9]([-a-zA-Z0-9]*[a-zA-Z0-9])?"
+const guidFmt string = "[a-zA-Z0-9]([-a-zA-Z0-9.]*[a-zA-Z0-9])?"
```

Comment:
why do we care since we don't do anything with this data except echo it back to the broker?